app/resources/upload_resource.rb

```diff
@@ -0,0 +1,9 @@
+class UploadResource < BaseResource
+  include ScopelessResource
+  attribute :content, format: :attachment
+  attributes :owner_id, :owner_type
```

Comment:
This should be `has_one :owner, polymorphic: true`, and you _might_ need an empty `OwnerResource` similar to our existing `ItemResource`.  Also needs a `has_many :uploads` on Comment and Post resources